files/qsub.pl

```diff
@@ -583,28 +583,28 @@ sub parse_script
         }
     };
 
-    # handle queues
-    if ($orig_argsh{'q'} || $set{'q'}) {
-        warn "Please do not set a queue (\'-q\' option or \'#PBS -q\' directive).\n" .
-            "Right now it sets only the default walltime based on the queue,\n" .
-            "So please request explicitly the desired walltime.\n";
-
-    }
+    # handle queues and special queues (as a partition)
     my $userqueue = $destination || $set{'q'};
     if ($userqueue) {
-        my %modwalltime = (
-            'short'  => '1:00:00',
-            'bshort' => '1:00:00', 
-            'long'   => '72:00:00',
-            'debug'  => '15:00',
-        );
-        if ($modwalltime{$userqueue}) {
-            splice(@newtxt, 1, 0, "## walltime from deprecated queue $userqueue", "#PBS -l walltime=$modwalltime{$userqueue}");
-        } else {
-            fatal("You have used a non-existing queue name!\n");
+        if ($userqueue =~ "_special") {
```

Comment:
to match, use `$userqueue =~ m/pattern/`

Reply:
but what's with the `_special`? users submit to the special queue, like `-q special`, and you have to map it to the `cluster_special` partition

Reply:
done